src/contracts/math/safe-math.sol

```diff
@@ -9,12 +9,13 @@ library SafeMath {
 
   /**
    * @dev Multiplies two numbers, reverts on overflow.
-   * @param _a Factor number.
-   * @param _b Factor number.
+   * @param factor1 Factor number.
+   * @param factor2 Factor number.
+   * @return The product of the two factors
    */
   function mul(
-    uint256 _a,
-    uint256 _b
+    uint256 factor1,
```

Comment:
Argument naming is not consistent with the rest of our code. Please add underscores before argument names. Example: `_factor1`.

Reply:
Updated at 70f0dcf0a2bd5d07a7c9977c2af365e977cb5862.

For a newer style guide, I'll argue against underscores everywhere. But not today.